llm-service/app/services/caii/utils.py

```diff
@@ -46,6 +47,9 @@ def build_auth_headers() -> Dict[str, str]:
 
 
 def get_caii_access_token() -> str:
+    if os.environ["CDP_TOKEN_OVERRIDE"]:
+        return os.environ["CDP_TOKEN_OVERRIDE"]
```

Comment:
This would raise an exception if `CDP_TOKEN_OVERRIDE` isn't in the environment at all! To be safer:

```suggestion
    if token_override := os.environ.get("CDP_TOKEN_OVERRIDE"):
        return token_override
```